packages/wranglerjs-compat-webpack-plugin/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "wranglerjs-compat-webpack-plugin",
+  "version": "0.0.0",
+  "description": "A webpack plugin to emulate the behavior of wrangler1's `type = webpack`",
+  "author": "wrangler@cloudflare.com",
+  "license": "MIT OR Apache-2.0",
+  "type": "commonjs",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "bugs": {
+    "url": "https://github.com/cloudflare/wrangler/issues"
+  },
+  "homepage": "https://github.com/cloudflare/wrangler#readme",
+  "scripts": {
+    "build": "run-p build:*",
+    "build:d.ts": "tsc",
+    "build:js": "esbuild src/index.ts --bundle --platform=node --external:execa --external:webpack --external:esbuild --external:rimraf --target=node16.7 --outfile=lib/index.js --format=cjs --sourcemap",
+    "test": "jest"
+  },
+  "dependencies": {
+    "esbuild": "^0.14.31",
+    "rimraf": "^3.0.2"
+  },
+  "devDependencies": {
+    "@octokit/core": "^3.6.0",
+    "@schemastore/package": "^0.0.6",
+    "@types/jest": "^27.4.1",
+    "@types/rimraf": "^3.0.2",
+    "@types/tar": "^6.1.1",
+    "@types/webpack": "^4.41.32",
+    "execa": "^6.1.0",
+    "jest": "^27.5.1",
+    "jest-fetch-mock": "^3.0.3",
+    "npm-run-all": "^4.1.5",
+    "tar": "^6.1.11",
+    "tosource": "^2.0.0-alpha.3",
+    "typescript": "^4.6.3",
+    "undici": "^5.0.0",
+    "webpack": "^4.46.0",
+    "wrangler": "../wrangler"
+  },
+  "peerDependencies": {
+    "webpack": "^4.46.0"
+  },
+  "jest": {
+    "restoreMocks": true,
+    "testTimeout": 180000,
+    "testRegex": ".*.(test|spec)\\.[jt]sx?$",
+    "transformIgnorePatterns": [
+      "node_modules/(?!find-up|locate-path|p-locate|p-limit|yocto-queue|path-exists|execa|strip-final-newline|npm-run-path|path-key|onetime|mimic-fn|human-signals|is-stream|get-port|supports-color)"
+    ],
+    "moduleNameMapper": {
+      "clipboardy": "<rootDir>/../wrangler/src/__tests__/helpers/clipboardy-mock.js",
+      "miniflare/cli": "<rootDir>/../../node_modules/miniflare/dist/src/cli.js"
+    },
```

Comment:
```suggestion
```
I don't think we need this in this package? might want to have a look at trimming `transformIgnorePatterns` as well

Reply:
Since we depend on `wrangler`, we actually need all of these :(